runtime/tests/jvmtitests/src/com/ibm/jvmti/tests/getConstantPool/gcp_bcdump.c

```diff
@@ -282,7 +282,11 @@ const char * const maciek_JavaBCNames[] = {
 "JBunimplemented" /* 240 */,
 "JBunimplemented" /* 241 */,
 "JBunimplemented" /* 242 */,
-"JBreturnToMicroJIT" /* 243 */,
+#if defined(J9VM_OPT_MICROJIT)
+"JBunimplemented" /* 243 */, /* The new MicroJIT does support partial compilation */
+#else
+"JBreturnToMicroJIT" /* 243 */, /* This line existed before the MicroJIT Rewrite, and looks to be a hold over from the old MicroJIT */
+#endif
```

Comment:
This is dead code under an `#if 0`.  I'd be more likely to delete the file than to update it :)

Reply:
I'll remove the update, but I'll leave the deletion for another PR/issue combo since this is already a pretty dense and large PR. Thanks for the heads up that this was dead though :smile: